src/patternfly/components/Label/label-content.hbs

```diff
@@ -0,0 +1,9 @@
+<{{# if label-content--IsLink}}a{{else}}span{{/if}} class="pf-c-label__content{{#if label-content--modifier}} {{label-content--modifier}}{{/if}}"
```

Comment:
Not necessarily. The `__content` element is the container for the icon and text, so we could just have the text inline with the icon. This gives us a container for the text so we have more control over it, though it's not necessary to have right now.

Reply:
hmm.. i think the structure should be something similar to the button component. since we decided not to add the text wrapper, i don't know if its necessary to have two wrappers around the text in the label component ( `pf-c-label` pf-c-label__content` and `pf-c-label__text` ) @mattnolting thoughts?

Reply:
works for me!

Reply:
@christiemolloy actually just looked at it again and one reason we use the text element is, similar to chips, it serves as a value passed to the close button's `aria-labelledby` attribute.